documentation/homework/week3/todo/src/components/Assign.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react'
+
+import InputField from './sub-components/InputField'
+
+class AssignItem extends Component {
+
+  render() {
+    //x -- 2x inputs
+    //x -- 1x button
+    return (
+      <form onSubmit={(event) => this.props.handleAssign(event)}>
+        <InputField name="description" />
+        <InputField name="deadline" />
```

Comment:
So you're using a regular text input here for a date field. It's not great, as I can type "blahblahblah" and then click add new todo, and I end up seeing UI that says "Invalid date". Please look into [HTML5 date inputs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date), or even check out 3rd party React datepicker components on npm.